[TR] Add Mappers' Guild (#3903)
ppy/osu-wiki: wiki/Mappers_Guild/tr.md

Loncanın organizasyonu ve tartışması [osu!dev Discord sunucusundaki](https://discord.gg/ppy) `#mappers-guild` kanalında bulunabilir.

## Sistem

*Mappers' Guild sistemi hakkında güncel bilgiye erişmek için [loncanın web sayfasındaki SSS bölümüne](https://mappersguild.com/faq) bakınız.*

3 ya da daha fazla dereceli mape sahip mapperlar loncaya istedikleri zaman [Mappers' Guild web sayfası](https://mappersguild.com/) üzerinden osu! hesaplarını doğrulatmak suretiyle katılabilirler. Diğer herkes ise web sayfasının görevlerini ve mapleme aktivitesini gözlemlemekte serbesttir.

Mapperlar Mappers' Guild'e bağlı olan featured artistlerin maplerinin derecelendirilme prosedürüne katkıda bulunarak ödüller kazanabilir. Bu, temel olarak beatmap oluşturmaya odaklanır, ancak [modlama](/wiki/Modding) yaparak, [storyboard](/wiki/Storyboards) oluşturarak, ya da lonca içindeki görevleri tamamlayarak katkıda bulunan kullanıcılara da ödül verilir.

### Partiler

> Partiler birlikte görev tamamlamak için kayıt yaptıran mapper gruplarıdır. Mapperlar tek seferde istedikleri sayıda partiye katılabilirler.

Reviewer: ```suggestion
Partiler birlikte görev tamamlamak için kayıt yaptıran mapper gruplarıdır. Mapperlar aynı anda istedikleri sayıda partiye katılabilirler.
```

> Özel görevlere katılmak üst lonca rütbesi gerektirir. Bir partinin üst bir lonca rütbesinden sayılması için, üyelerinin en az yarısının gerekli rütbe eşiti ya da üstü olması gereklidir.

Reviewer: ```suggestion
Özel görevlere katılmak için yüksek lonca rütbesi gerekir. Bir partinin yüksek lonca rütbesinde sayılması için, üyelerinin en az yarısının gerekli rütbede ya da daha yüksekte olması gereklidir.
```

### Görevler

Görevler, mapperlar featured artist mapleri oluştururken belirli gereksinimleri sağladıklarında onları ödüllendirir. Görevler genellikle yeni bir featured artist için belirli bir sayıda beatmap oluşturmayı kapsar, ancak bazı görevler ufak numaralar gerektirir.

> Her bir görevin parti büyüklüğü/rütbesi şartları, kabul etme bedeli, ve bitiş tarihi vardır. Bir partinin bütün üyeleri görevi kabul etmek için belirtilen bedeli ödemek zorundadır. Bu bedel genellikle tek kişilik görevler ve numaralar gerektiren görevler için daha büyüktür.

Reviewer: should gimmick really be translated as `numara`?

Görevlerin süresi 3 aylık inaktiflik sonrasında dolar. Herhangi bir kullanıcı mevcut Mappers' Guild puan bakiyesinden harcayarak görevleri geri açabilir.

> Mappers' Guild için görevlere katılım zorunlu değildir. Eğer isterse bir kullanıcı tek başına beatmapler oluşturarak ödüller kazanabilir.

Reviewer: ```suggestion
Mappers' Guild için görevlere katılım zorunlu değildir. Bir kullanıcı, eğer isterse, sadece beatmapler oluşturarak ödüller kazanabilir.
```

Author: ```suggestion
Mappers' Guild için görevlere katılım zorunlu değildir. Bir kullanıcı, eğer isterse, beatmapleri kendi başına oluşturarak ödüller kazanabilir.
```
your suggestion alters the meaning meant to be given here, this should be better ^

### Ödüller

Ödüller bir puan sistemi aracılığı ile dağıtılır. Bir kullanıcı belirli bir puan eşiğine vardığında, bir profil rozeti ile belirtilen bir rütbe elde ederler. Sırasıyla 100, 250 ve 500 puanda olmak üzere üç farklı profil rozeti seviyesi vardır.

> Ödüller kullanıcının toplam puanlarına dayalıdır. Bir kullanıcının mevcut puanları osu!'nun [kudosu](/wiki/Modding/Kudosu) "toplam kazanılan" ve "mevcut" puan havuzları ile aynı modeli takip eden onun toplam puanlarının bir alt sekmesidir.

Reviewer: ```suggestion
Ödüller kullanıcının toplam puanına dayalıdır. Bir kullanıcının mevcut puanı ise, osu!'nun [kudosu](/wiki/Modding/Kudosu) "toplam kazanılan" ve "mevcut" puan havuzları gibi, toplam puanlarının bir kısmıdır.
```
`subsection` is hard to translate. maybe we should do a bigger change here

Author: ```suggestion
Ödüller kullanıcının toplam puanına dayalıdır. Bir kullanıcının mevcut puanı ise, osu!'nun [kudosu](/wiki/Modding/Kudosu) "toplam kazanılan" ve "mevcut" puan havuzları gibi, toplam puanlarının bir yan havuzudur.
```
this should do I think

> Puanlar maplemeyle alakalı ödevleri tamamlayarak kazanılır. Kazanılan miktar ödevler arasında değişiklik gösterir, ancak genellikle harcanması gereken zamana oranlıdırlar. Örneğin, `Hard` bir zorluğu maplemek `Easy` bir zorluğu maplemekten daha uzun zaman alır, bundan ötürü ilki ikinciden daha fazla puan verir.

Reviewer: ```suggestion
Puan, maplemeyle alakalı görevleri tamamlayarak kazanılır. Kazanılan miktar görevler arasında değişiklik gösterir, ancak genellikle harcanması gereken zamana doğru orantılıdır. Örneğin, `Hard` bir zorluğu maplemek `Easy` bir zorluğu maplemekten daha uzun zaman alır, bundan ötürü ilki ikinciden daha fazla puan verir.
```

> Eğer beatmap bir göreve bağlıysa zorluk oluşturmak kullanıcıya daha fazla puan kazandırır. Bir görevi bitiş tarihinden önce tamamlamak kullanıcılara maplenen her bir zorluk için bonus puan ve göreve dahil olan tüm kullanıcılara eşit miktarda bonus puan verir. Bu eşit miktardaki bonus bitiş tarihi geçtikten sonra kaybedilir, ancak zorluk bonusları azalan getirileriyle birlikte bitiş tarihinden sonraki 0, 20, ve 40 günlük artışlarıyla korunur.

Reviewer: ```suggestion
Eğer beatmap bir göreve bağlıysa zorluk oluşturmak kullanıcıya daha fazla puan kazandırır. Bir görevi bitiş tarihinden önce tamamlamak, kullanıcılara maplenen her bir zorluk için bonus puan ve göreve dahil olan tüm kullanıcılara eşit miktarda bonus puan verir. Bu eşit miktardaki bonus bitiş tarihi geçtikten sonra kaybedilir, ancak zorluk bonusları bitiş tarihinden sonraki 0, 20, ve 40'ıncı gün gittikçe azalarak korunur.
```

## Tarihçe

Mappers' Guild 10 mapperdan daha az bir grup ile 2017'nin sonlarına doğru oluşturuldu. O dönemde yeni eklenecek olan bir featured artist, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), hakkındaki ön bilgilendirme ile sanatçının [haber duyurusuna](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) eklenmek üzere onlar 5 tane beatmapin oluşturulması için görevlendirildi.

Daha önceki featured artistleri tanıtmak için, grup üye sayısını arttırdı ve farklı sanatçılardan 8 tane beatmap derecelendirdi. Bu beatmapler indirilebilir bir beatmap paketiyle birlikte bir [haber gönderisi](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) ile yayımlandı.

> Her iki beatmap paketinin ayak izlerini takip ederek, Mappers' Guild, daha eski featured artist şarkılarının paketlerine ek olarak, [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), ve [HyuN](https://osu.ppy.sh/beatmaps/artists/33) gibi, yeni featured artist duyuruları için paketler yayımlarken üye sayısını da sürekli olarak arttırdı.

Reviewer: `ayak izlerini takip ederek` not really a thing in turkish

### Otomasyon

Mappers' Guild için başlangıçtaki plan görevler gibi MMO elementleri bulundurmak olsa da, belirli sayıdaki lonca katılımcıları için bitiş tarihlerini manuel olarak idare etmenin oldukça elverişsiz olduğu belirlendi. 2018'in sonlarına doğru, ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) ve ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) Mappers' Guild sistemini otomatikleştirecek, ödüllerin paylaştırılmasını ve görev/parti sistemlerinin yönetimini daha elverişli hale getirecek bir websitesi geliştirmeye başladı.

> Mappers' Guild ile ilgili her şeyin günümüzde kontrol edildiği yer olan [Mappers' Guild websitesi](https://mappersguild.com/), 1 Mart 2019 tarihinde faaliyete geçti. Herkese açık olduğundan dolayı, lonca 500'ün üzerinde bir üye sayısına ulaşmış ve osu! topluluğunun geri kalanını featured artist mapleme içerikleriyle güncel tutmak amacıyla [güncelleme haber gönderilerini](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) yayımlamaktadır.

Reviewer: ```suggestion
Mappers' Guild ile ilgili her şeyin günümüzde kontrol edildiği yer olan [Mappers' Guild websitesi](https://mappersguild.com/), 1 Mart 2019 tarihinde faaliyete geçti. Herkese açık olduğu için, lonca 500'ün üzerinde bir üye sayısına ulaşmıştır ve osu! topluluğunun geri kalanını featured artist mapleme içerikleriyle güncel tutmak amacıyla [güncelleme haber gönderilerini](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) yayımlamaktadır.
```

i still don't like the last sentence too much. not sure on a better translation though
